BCH-Wunder-Project-team-4/Wunder.io
# Listing events

## Link to ticket:

https://trello.com/c/SWcFlwyk/89-content-type-event

## Link to feature environment:

http://localhost:3000/

## Changes proposed in this PR:

Now you can create a listing of events which is a simple paragraph type. 
Recommended number of elements in list is 3

![Screenshot 2023-12-08 at 12 56 02](https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/117649417/0e36792e-ed87-4caa-9073-0e19fb8f3655)

## How to test:

1. Commit your changes
2. lando drush cex
3. pull
4. lando drush cim
5. create a few events if you don't have any
6. add a listing events paragraph to any page
7. check the front 

## Best practices:

<details>
<summary><h3>Accessibility:</h3></summary>
<p>
This project must support WCAG accessibility level AA <em>(edit this according to the requirements of your project)</em>. To ensure this standard is met, remember to:

- Perform automated checks using a tool such as Wave or SiteImprove.
- Test keyboard navigation: are all parts of the UI navigable using only the keyboard? Is the tab order logical? Can popups, menus etc be dismissed with the escape key?
- Test responsiveness, scaling and text reflow.
- Make sure no accessibility issues exist on either desktop or mobile views.
- If you have time, test with a screen reader such as VoiceOver (macOS), NVDA (Windows), or Orca (Linux).

Use the [Accessibility Testing Cheat Sheet](https://intra.wunder.io/info/accessibility-group/accessibility-testing-cheat-sheet) for information on how to run these tests.
</p>
</details>
